test/ReadsUtils_server_test.py

```diff
@@ -1050,22 +1147,44 @@ def test_upload_fail_interleaved_for_single(self):
              'fwd_id': ret['id'],
              'name': 'bar'
              },
-            'Invalid fasta file /kb/module/work/tmp/fwd/Sample5_interleaved' +
-            '.fastq from Shock node ' + ret['id'])
+            'Invalid fasta file /kb/module/work/tmp/fwd/Sample5_interleaved.fastq')
```

Comment:
There is no shock id since there is no longer "process_shock_ids_for_upload".
They are now local files because we did stats on them and potentially interleaved them.